test/index.js

```diff
@@ -237,8 +280,11 @@ describe('simple-keyring', function () {
       },
     ];
 
-    it('returns the expected value', async function () {
+    beforeEach(async function () {
```

Comment:
Similar for here — mind copying this into each test that needs it?

Reply:
yep changed